src/model.ts

```diff
@@ -547,9 +551,12 @@ export class Singleton extends $Module {
   superclass(): Class {
     const superclassReference = this.supertypes.find(supertype => supertype.reference.target()?.is('Class'))?.reference
     if(superclassReference) return superclassReference.target() as Class
-    else return this.environment().getNodeByFQN<Class>('wollok.lang.Object')
+    else return this.objectClass
   }
 
+  isClosure(parametersCount = 0): boolean {
+    return this.methods().some(method => method.name === '<apply>' && method.parameters.length === parametersCount)
```

Comment:
Creo que sería mejor usar: `!!this.lookupMethod('<apply>', parametersCount)`, en parte porque es más breve y en parte porque ya considera los varargs y esas cosas.

Reply:
👌🏼